Lecture8/src/main/java/pageObject/TractorPricesPage.java

```diff
@@ -0,0 +1,30 @@
+package pageObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class TractorPricesPage extends AbstractPage {
+
+    CatalogTractorsPage catalogTractorsPage = new CatalogTractorsPage(driver);
+
+    @FindBy(xpath = "//span[contains(.,'По цене')]/..")
+    private WebElement sortOffersByPriceLink;
+
+    @FindBy(xpath = "//td/a[@data-href='https://cart.onliner.by']")
+    private List<WebElement> listAddToCartLinks;
+
+    public TractorPricesPage(WebDriver driver) {
+        super(driver);
+        PageFactory.initElements(this.driver, this);
+    }
+
+    public void addTractorToCart() throws InterruptedException {
+        sortOffersByPriceLink.click();
+        Thread.sleep(2000);
+        listAddToCartLinks.get(catalogTractorsPage.getSelectedRandomNumber()).click();
+    }
```

Comment:
add method getTractorName() that returns parsed name of tractor

Reply:
added